client/my-sites/post-type-list/post-actions-ellipsis-menu/duplicate.jsx

```diff
@@ -25,12 +25,14 @@ function PostActionsEllipsisMenuDuplicate( {
 	translate,
 	canEdit,
 	status,
+	type,
 	duplicateUrl,
 	bumpStat,
 } ) {
 	const validStatus = includes( [ 'draft', 'future', 'pending', 'private', 'publish' ], status );
+	const validType = includes( [ 'post' ], type );
 
-	if ( ! isEnabled( 'posts/post-type-list' ) || ! canEdit || ! validStatus ) {
+	if ( ! isEnabled( 'posts/post-type-list' ) || ! canEdit || ! validStatus || ! validType ) {
```

Comment:
This should just be `type !== 'post'`, and we should get rid of the `validType` variable - if we support more types later on, we can update the code accordingly.